src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -170,38 +188,71 @@ public Object createModule(String name) throws CheckstyleException {
 
     /**
      * Create object with the help of the supplied map.
-     * @param name name of module.
-     * @param map the supplied map.
-     * @return instance of module if it is found in modules map.
-     * @throws CheckstyleException if the class fails to instantiate.
+     * @param name name of module
+     * @param map the supplied map
```

Comment:
I see there are no period in other javadoc tag in `PackageObjectFactory`, so I change this to keep them in a same style. If you don't agree with it I would change it back.